vnext/package.json

```diff
@@ -69,5 +69,57 @@
       "minor",
       "patch"
     ]
-  }
+  },
+  "files": [
+    
+    "/Chakra",
+    "/Common",
+    "/DeforkingPatches",
+    "/etc",
+    "/flow",
+    "/Folly",
+    "/include",
+    "/jest",
+    "/JSI",
+    "/Libraries",
+    "!/Libraries/**/*.windesktop.*",
+    "/local-cli",
+    "/Microsoft.ReactNative",
+    "/Microsoft.ReactNative.Cxx",
+    "/Microsoft.ReactNative.Managed",
+    "/Microsoft.ReactNative.SharedManaged",
+    "/Mso",
+    "/PropertySheets",
+    "/ReactCommon",
+    "/ReactUWP",
+    "/ReactWindowsCore",
+    "/Scripts",
+    "/Shared",
+    "/stubs",
+    "/Test",
+    "/V8Inspector",
+
+    "/.clang-format",
+    "/.editorconfig",
+    "/.flowconfig",
+    "/Directory.Build.props",
+    "/Directory.Build.targets",
+    "/index.js",
+    "/index.windows.d.ts",
+    "/index.windows.d.ts.map",
+    "/index.windows.js",
+    "/index.windows.js.map",
+    "/interface.js",
```

Comment:
>interface [](start = 6, length = 9)

I haven't figured out where these are created or used. 
They were in 0.62-preview.2 so I included them here.

Reply:
These are created during the JS build. We run tsc in src then copy everything to the root directory to match the layout of the normal react-native package, where these are just source files with no transformations. Having targets living with source is super gross, but necessary with currently limitations of metro.